support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -175,13 +200,40 @@ function OneTimeCheckoutComponent({
 	const [amount, setAmount] = useState<number | 'other'>(defaultAmount);
 	const [otherAmount, setOtherAmount] = useState<string>('');
 	const [otherAmountError, setOtherAmountError] = useState<string>();
-
-	const finalAmount = amount === 'other' ? parseFloat(otherAmount) : amount;
+	const finalAmount =
+		amount === 'other'
+			? Number.isNaN(parseFloat(otherAmount))
+				? undefined
+				: parseFloat(otherAmount)
+			: amount;
+
+	useEffect(() => {
+		if (finalAmount) {
+			setStripeExpressCheckoutEnable(true);
+			elements?.update({ amount: finalAmount * 100 });
+		} else {
+			// no onclick?
+			setStripeExpressCheckoutEnable(false);
+		}
+	}, [finalAmount]);
 
 	/** Payment methods: Stripe */
 	const stripe = useStripe();
 	const elements = useElements();
 	const cardElement = elements?.getElement(CardNumberElement);
+	const [, setStripeExpressCheckoutPaymentType] =
```

Comment:
We don't seem to be using this payment type, not sure if we should

Reply:
I think this might be used to inidicate `ApplePay` amount items within Stripe by checking `stripeExpressCheckoutPaymentType = 'apple-pay` and labelling the stripe paymentMethod as `StripeApplePay`. Annoyingly need to test this on CODE. Perhaps we can my last commit together to see if makes sense tomorrow morning?